### ISSUE TITLE
Feature request: Vertical orientation option

### ISSUE THREAD

**Issue by christianboyle** [jayfreestone/priority-plus#20]
First, thanks for this lib, it's super useful and amazing that it takes into account accessibility concerns.

Second, I'm working with a navigation layout that changes its orientation from horizontal (mobile) to vertical (desktop) based on viewport width.

Quick devtools prototyping on your demo page makes it seem like I can achieve this with some Flexbox customization, thanks to your use of `IntersectionObserver`.

I'm curious if you have any thoughts on this approach and figured I'd raise it as a possible feature request since it seems like a cool option to have. 

https://user-images.githubusercontent.com/1605754/178345072-654793c4-b912-4ba1-a80d-7c70a00b2116.mp4

**Comment by jayfreestone** [comment]
Thanks for your interest in the lib! Vertical support is definitely something I've considered in the past.

As you mentioned, it should be possible to just with just CSS. I had a quick play around and got it working, as you have:

https://user-images.githubusercontent.com/3719745/178526077-05f42488-4987-4483-aa65-cf5d5281d629.mp4


I've never really thought about a menu switching orientation like in your example, though. Neat!

Outside of potentially providing an additional stylesheet with better vertical defaults (and docs), is there anything else you see as barrier to vertical support? I think for the most part it just works, which is a happy accident 🎉.

**Comment by christianboyle** [comment]
@jayfreestone Any chance you could share what styles you used to achieve the effect in your screencap?

**Comment by jayfreestone** [comment]
Sure thing. These were the additions I made:

```css
.p-plus__primary {
  flex-direction: column;
}

.p-plus-container {
  display: flex;
}

.p-plus {
  height: 100vh;
  flex-direction: column;
}

.p-plus__overflow {
  left: 100%;
  bottom: 0;
  top: unset;
}
```

**Comment by christianboyle** [comment]
@jayfreestone Many thanks for those styles. For some reason I wasn't able to get things working properly yesterday, but with your styles and a little additional tweaking I managed to get the desired effect.

Thanks again, you saved me many hours of frustration! Much appreciated.

https://user-images.githubusercontent.com/1605754/178776576-71837cff-a0fc-4748-82a9-3ad817f0f3ca.mp4

I guess one parting thought is that it might be nice to have a static demo/example within the repo. Currently, the Glitch pen source links are broken, so I had to start with the demo site source and strip it down to the bare minimum to work on my proof of concept.
